master/static/srv/get_model_versions.sql

```diff
@@ -10,19 +14,21 @@ WITH mv AS (
     labels,
     notes,
     username,
+    user_version,
     last_updated_time
   FROM model_versions
-  LEFT JOIN users ON users.id = model_versions.user_id
+  LEFT JOIN u ON u.id = model_versions.user_id
   WHERE model_id = $1
 ),
 m AS (
-  SELECT m.id, m.name, m.description, m.notes, m.metadata, m.creation_time, m.last_updated_time, array_to_json(m.labels) AS labels, u.username, m.archived, COUNT(mv.version) as num_versions
+  SELECT m.id, m.name, m.description, m.notes, m.metadata, m.creation_time, m.last_updated_time, array_to_json(m.labels) AS labels,
+    username, user_version, m.archived, COUNT(mv.version) as num_versions
   FROM models as m
-  JOIN users as u ON u.id = m.user_id
+  JOIN u ON u.id = m.user_id
   LEFT JOIN model_versions as mv
     ON mv.model_id = m.id
   WHERE m.id = $1
-  GROUP BY m.id, u.id
+  GROUP BY m.id, u.id, username, user_version
```

Comment:
is there any point to have username and user_version here? `m.username` and `m.user_version` won't be used anywhere.

and then `u` CTE won't be reused anywhere, and so it's not needed, which is potentially better for optimizer.